pkg/sql/logictest/testdata/logic_test/information_schema

```diff
@@ -517,6 +516,13 @@ query T
 SET DATABASE = test; SELECT table_name FROM information_schema.tables WHERE table_schema = 'other_db'
 ----
 
+# Check that tables from other schemas are found
+query TT rowsort
+CREATE SCHEMA myschema; CREATE TABLE myschema.abc(); CREATE TABLE abc(); SELECT table_schema, table_name FROM information_schema.tables WHERE table_name = 'abc'
```

Comment:
Does this have to be 1 line?

Reply:
looks like they don't -- i was just using the pattern in the test cases above. updated this example